index.js

```diff
@@ -147,6 +157,10 @@ module.exports = plugin(
           DEFAULT: 0,
           ...theme("scale"),
         }),
+        animationRotate: ({ theme }) => ({
+          DEFAULT: 0,
```

Comment:
The default should be some reasonable amount that looks okay in many cases. It'd be weird to add `spin-in` and have nothing happen. 

I think 30deg would probably be a pretty good default.